### DISCUSSION TITLE
how to recover from a forgotten master password.

### DISCUSSION THREAD

**Discussion by kkarlekar** [kingToolbox/WindTerm#1445] [General]
I don't remember setting a master password on my macbook version of windterm, but a blank master password comes out as incorrect. How do I reset the master password without knowing the old one. 
It is fine, if i'm not able to access the encrypted data that was encrypted using the current master password

**Comment by kkarlekar** [comment]
found ~/.wind/profiles/default.v10 and removed in user.config file  the configuration items application.fingerprint and application.masterPassword

**Comment by wedlex** [comment]
I have a folder here `/System/Volumes/Data/Users/Shared/windterm/.wind`
if you can't find it use search in terminal: `find / -name ".wind"`
